src/cf-layout/usage.md

```diff
@@ -1230,34 +1230,93 @@ Featured content module, like a hero, consists of headline/text/optional call to
+
+<section class="block block__border block__flush o-featured-content-module">
+    <div class="o-featured-content-module_text">
+        <div class="category-slug">
+            <span class="o-featured-content-module_icon cf-icon cf-icon-speech-bubble">
+</span> Featured (Map)</div>
+        <h2>Feature title</h2>
+        <p>Lorem ipsum dolor sit amet, ei ius adhuc inani iudico, labitur instructior ex pri. Cu pri inani constituto, cum aeque noster commodo cu.</p>
+        <a class="jump-link jump-link__underline"> <span class="jump-link_text">Read more about the feature</span> </a>
+    </div>
+    <div class="o-featured-content-module_visual"> <img class="o-featured-content-module_img o-featured-content-module_img__map" src="http://placekitten.com/g/540/270"> </div>
+</section>
+
+```
+<section class="block block__border block__flush o-featured-content-module">
+    <div class="o-featured-content-module_text">
+        <div class="category-slug">
+            <span class="o-featured-content-module_icon cf-icon cf-icon-speech-bubble">
+</span> Featured (Map)</div>
+        <h2>Feature title</h2>
+        <p>Lorem ipsum dolor sit amet, ei ius adhuc inani iudico, labitur instructior ex pri. Cu pri inani constituto, cum aeque noster commodo cu.</p>
+        <a class="jump-link jump-link__underline"> <span class="jump-link_text">Read more about the feature</span> </a>
+    </div>
+    <div class="o-featured-content-module_visual"> <img class="o-featured-content-module_img o-featured-content-module_img__map" src="http://placekitten.com/g/540/270"> </div>
+</section>
+```
+
+### Featured content module - Videos
+
+When the featured content module image is a video, the `o-featured-content-module_img__video` modifier class is added to the `o-featured-content-module_img` image. This anchors the center of the image to the center of the available space so that the focal point of the video generally remains centered.
+
+<section class="block block__border block__flush o-featured-content-module">
+    <div class="o-featured-content-module_text">
+        <div class="category-slug">
+            <span class="o-featured-content-module_icon cf-icon cf-icon-speech-bubble">
+</span> Featured (Video) </div>
+        <h2>Feature title</h2>
+        <p>Lorem ipsum dolor sit amet, ei ius adhuc inani iudico, labitur instructior ex pri. Cu pri inani constituto, cum aeque noster commodo cu.</p>
+        <a class="jump-link jump-link__underline"> <span class="jump-link_text">Read more about the feature</span> </a>
+    </div>
+    <div class="o-featured-content-module_visual"> <img class="o-featured-content-module_img o-featured-content-module_img__video" src="http://placekitten.com/g/540/270"> </div>
+</section>
+
+```
+<section class="block block__border block__flush o-featured-content-module">
```

Comment:
Oh yeah, we've had a world of hurt when combining atomic elements with layout classes like block. Sorry this never got updated before, but we should probably figure out how to break them apart now. It could probably happen in a separate PR as well to keep things moving.